pkg/printers/internalversion/printer.go

```diff
@@ -256,7 +258,7 @@ func printTemplateList(list *templateapi.TemplateList, w io.Writer, opts kprinte
 	return nil
 }
 
-func printBuild(build *buildapi.Build, w io.Writer, opts kprinters.PrintOptions) error {
+func printBuild(build *buildv1.Build, w io.Writer, opts kprinters.PrintOptions) error {
```

Comment:
This is questionable.  Make a followup issue to revert this part, duplicate, and separate.

Reply:
+1 for the issue

Reply:
actually, not needed, I reverted the changes here and used the internal helpers instead